pmm-app/src/pmm-check/components/Table/TableBody.test.tsx

```diff
@@ -9,7 +9,7 @@ describe('TableBody::', () => {
   it('should render a table body with 3 rows', () => {
     const root = shallow(<TableBody data={activeCheckStub} />);
 
-    expect(root.find('tbody > tr').length).toEqual(4);
+    expect(root.find('tbody > tr').length).toEqual(5);
```

Comment:
`should render a table body with 3 rows` -> why is the test asserting 5 `tr` ?

Reply:
Removed the too specific text